### DISCUSSION TITLE
how do i get a foreign pluto tv live broadcast and get a m3u link from it

### DISCUSSION THREAD

**Discussion by whyiyeet** [iptv-org/community#466] [Q&A] [answered]
so i wanna make a m3u file/link for a channel on pluto tv and idk what to do also im trying to get one from germany where i dont live. can anybody help me out with getting a foreign pluto tv broadcast and having it put in a m3u link so i can watch the channel on vlc?

**Comment by PopeyeTheSai10r** [comment, marked as answer]
https://github.com/Free-TV/IPTV/blob/master/playlists/playlist_germany.m3u8 may have what you are looking for.

**Reply by whyiyeet** [reply]
thanks
